### ISSUE TITLE
[Security issue] [CERT-FR] CERTFR-2020-AVI-301

### ISSUE THREAD

**Issue by aguerson** [bigbluebutton/greenlight#1633]
https://www.cert.ssi.gouv.fr/avis/CERTFR-2020-AVI-301/

From my Gemfile and Gemfile.lock

`$ diff Gemfile.lock Gemfile.lock_CERTFR-2020-AVI-301`

```
14,15c14,15
<     actioncable (5.2.3)
<       actionpack (= 5.2.3)
---
>     actioncable (5.2.4.3)
>       actionpack (= 5.2.4.3)
18,21c18,21
<     actionmailer (5.2.3)
<       actionpack (= 5.2.3)
<       actionview (= 5.2.3)
<       activejob (= 5.2.3)
---
>     actionmailer (5.2.4.3)
>       actionpack (= 5.2.4.3)
>       actionview (= 5.2.4.3)
>       activejob (= 5.2.4.3)
24,27c24,27
<     actionpack (5.2.3)
<       actionview (= 5.2.3)
<       activesupport (= 5.2.3)
<       rack (~> 2.0)
---
>     actionpack (5.2.4.3)
>       actionview (= 5.2.4.3)
>       activesupport (= 5.2.4.3)
>       rack (~> 2.0, >= 2.0.8)
31,32c31,32
<     actionview (5.2.3)
<       activesupport (= 5.2.3)
---
>     actionview (5.2.4.3)
>       activesupport (= 5.2.4.3)
37,38c37,38
<     activejob (5.2.3)
<       activesupport (= 5.2.3)
---
>     activejob (5.2.4.3)
>       activesupport (= 5.2.4.3)
40,44c40,44
<     activemodel (5.2.3)
<       activesupport (= 5.2.3)
<     activerecord (5.2.3)
<       activemodel (= 5.2.3)
<       activesupport (= 5.2.3)
---
>     activemodel (5.2.4.3)
>       activesupport (= 5.2.4.3)
>     activerecord (5.2.4.3)
>       activemodel (= 5.2.4.3)
>       activesupport (= 5.2.4.3)
46,48c46,48
<     activestorage (5.2.3)
<       actionpack (= 5.2.3)
<       activerecord (= 5.2.3)
---
>     activestorage (5.2.4.3)
>       actionpack (= 5.2.4.3)
>       activerecord (= 5.2.4.3)
50c50
<     activesupport (5.2.3)
---
>     activesupport (5.2.4.3)
211,220c211,220
<     rails (5.2.3)
<       actioncable (= 5.2.3)
<       actionmailer (= 5.2.3)
<       actionpack (= 5.2.3)
<       actionview (= 5.2.3)
<       activejob (= 5.2.3)
<       activemodel (= 5.2.3)
<       activerecord (= 5.2.3)
<       activestorage (= 5.2.3)
<       activesupport (= 5.2.3)
---
>     rails (5.2.4.3)
>       actioncable (= 5.2.4.3)
>       actionmailer (= 5.2.4.3)
>       actionpack (= 5.2.4.3)
>       actionview (= 5.2.4.3)
>       activejob (= 5.2.4.3)
>       activemodel (= 5.2.4.3)
>       activerecord (= 5.2.4.3)
>       activestorage (= 5.2.4.3)
>       activesupport (= 5.2.4.3)
222c222
<       railties (= 5.2.3)
---
>       railties (= 5.2.4.3)
233,235c233,235
<     railties (5.2.3)
<       actionpack (= 5.2.3)
<       activesupport (= 5.2.3)
---
>     railties (5.2.4.3)
>       actionpack (= 5.2.4.3)
>       activesupport (= 5.2.4.3)

```

`$ diff Gemfile Gemfile_CERTFR-2020-AVI-301`

```
11c11,12
< gem 'rails', '~> 5.2.3'
---
> # CERTFR-2020-AVI-301
> gem 'rails', '~> 5.2.4.3'

```

**Comment by farhatahmad** [comment]
Hi @aguerson,

Thanks for reporting this. We'll get Rails update to 5.2.4.3 in the next release

**Comment by farhatahmad** [comment]
Fixed in v2.6
